src/js/components/Ballot/OfficeItemCompressed.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { Component, PropTypes } from "react";
+import { Link, browserHistory } from "react-router";
+import StarAction from "../../components/Widgets/StarAction";
+import SupportStore from "../../stores/SupportStore";
+import { capitalizeString } from "../../utils/textFormat";
+
+export default class OfficeItemCompressed extends Component {
+  static propTypes = {
+    key: PropTypes.string,
+    we_vote_id: PropTypes.string.isRequired,
+    kind_of_ballot_item: PropTypes.string.isRequired,
+    ballot_item_display_name: PropTypes.string.isRequired,
+    link_to_ballot_item_page: PropTypes.bool,
+  };
+  constructor (props) {
+    super(props);
+    this.state = {transitioning: false};
+  }
+
+  componentDidMount () {
+    this.supportStoreListener = SupportStore.addListener(this._onChange.bind(this));
+    this.setState({ supportProps: SupportStore.get(this.props.we_vote_id) });
+  }
+
+  componentWillUnmount () {
+    this.supportStoreListener.remove();
+  }
+
+  _onChange () {
+    this.setState({ supportProps: SupportStore.get(this.props.we_vote_id), transitioning: false });
+  }
+  render () {
+    const { supportProps, transitioning } = this.state;
+    let { ballot_item_display_name, we_vote_id } = this.props;
+    let officeLink = "/office/" + we_vote_id;
+    let goToOfficeLink = function () { browserHistory.push(officeLink); };
+
+    ballot_item_display_name = capitalizeString(ballot_item_display_name);
+
+    return <div className="office-card__container">
+      <div className="office-card">
+        <div className="office-card__media-object">
+          <div className="office-card__media-object-anchor">
+
+          </div>
+          <div className="office-card__media-object-content">
+            {
```

Comment:
We shouldn't need information about support/oppose in network for Offices; while users can comment on an office using the ItemPositionStatementActionBar, we aren't allowing them to support or oppose the office itself